test/gululog_topic_SUITE.erl

```diff
@@ -91,6 +92,34 @@ t_basic_flow(Config) when is_list(Config) ->
   ok = gululog_r_cur:close(C5),
   ok.
 
+t_flush({init, Config}) -> Config;
+t_flush({'end', _Config}) -> ok;
+t_flush(Config) when is_list(Config) ->
+  Dir = ?config(dir),
+  T0 = gululog_topic:init(Dir, [{segMB, 1}]),
+  T1 = gululog_topic:append(T0, <<"header1">>, <<"body1">>),
+  T2 = gululog_topic:append(T1, <<"header2">>, <<"body2">>),
+
+  ?assertMatch(empty, gululog_r_cur:open(Dir, 0)),
```

Comment:
is it reliable ?
what if this common test process is some how scheduled out by scheduler, then resumed after the delayed writes actually happened?